polyfill/lib/casts.mjs

```diff
@@ -0,0 +1,140 @@
+import { ES } from './ecmascript.mjs';
+
+export function absolute(arg) {
+  const Absolute = ES.GetIntrinsic('%Temporal.Absolute%');
+  if (arg instanceof Absolute) return arg;
```

Comment:
instanceof seems OK for a polyfill, but presumably the spec will be based on internal slots (and therefore cross-realm).

Reply:
Am I wrong in wanting:

 a) cross-realm to cast to a current-realm object
 b) current-realm to be a no-op

???

Reply:
#165